onboarding/guide-to-getting-started.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Guide to getting started
+description: A list of things to complete when you start
+---
+
+## Your first day
+
+_Goal: I’ve met my manager and some members of the engineering team (over lunch at Tataki). I’ve logged into slack, google (artsymail), 1Password, and know how to use the coffee machine._
+
+## Your first week
+
+_Goal: I can ssh into gravity (staging and production). I can PR to a repo in Artsy’s Github org. I know what the engineering teams are, what their main focuses are, and who leads them. I know my way around the Artsy site._
+
+### First week checklist
+
+#### Dev environment:
+
+- [ ] Get invited to the Artsy Github org
+  _Ask your mentor or another engineer for help, or log into github using the it@artsymail.com account in 1Password and add yourself._
```

Comment:
We might need more detailed instructions or clarifications for the new Github (and any other) https/token/12factor stuff.